src/interfaces/token.ts

```diff
@@ -113,6 +113,35 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       try {
         const zapperBalances = await this.yearn.services.zapper.balances(account);
         balances.zapper = zapperBalances.filter(({ address }) => addresses.zapper.has(address));
+        const balance = await this.ctx.provider.read.getBalance(account);
+        const priceUsdc = await this.yearn.services.oracle.getPriceUsdc(EthAddress);
```

Comment:
Should we move all this logic after the `try...catch`? If Zapper fails, we want to still return ETH balance.

Reply:
solved on fd85573125e7eda48038fbb369ac4c7b2f93d0e0